frame/staking/src/pallet/impls.rs

```diff
@@ -1277,8 +1279,10 @@ impl<T: Config> VoteWeightProvider<T::AccountId> for Pallet<T> {
 /// does not provided nominators in sorted ordered. If you desire nominators in a sorted order take
 /// a look at [`pallet-bags-list].
 pub struct UseNominatorsMap<T>(sp_std::marker::PhantomData<T>);
+
 impl<T: Config> SortedListProvider<T::AccountId> for UseNominatorsMap<T> {
 	type Error = ();
+	type Value = VoteWeight;
```

Comment:
I think the type name could be better? at least more descriptive than "value"

Reply:
I discussed this for a little while with Kian, naming is hard :) I was only thinking something like OrderableType, but then things like `update_orderable_type_of` start sounding funny. I was trying to think of a word that indicates the type can be ordered/compared

Reply:
`Score`?

Reply:
I like that, changed it to `Score`